accounts/abi/bind/backends/simulated_test.go

```diff
@@ -693,7 +686,7 @@ func TestHeaderByNumber(t *testing.T) {
 func TestTransactionCount(t *testing.T) {
 	testAddr := crypto.PubkeyToAddress(testKey.PublicKey)
 
-	sim := simTestBackend(testAddr)
+	sim := simTestBackend(t, core.GenesisAlloc{testAddr: {Balance: big.NewInt(1e18)}})
```

Comment:
Seems like maybe it had been better to leave the `simTestBackend` as it was -- it's only a helper method for testing. Now a lot of boilerplate code suddenly becomes a lot clunkier

Reply:
sure  I can revert, is that the only change that is requered?

Reply:
>  is that the only change that is requered?

I couldn't say. A PR review can sometimes be a lengthy affair

Reply:
I meant is that the only change required for your review?
 Ideally, I would like to make the changes in bulk rather than one by one.

Reply:
your time is valuable but mine is valuable too, I am sorry as well.